src/serverError.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable max-len */
+const ERROR_MESSAGE_NO_TEXT = 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
```

Comment:
This part `Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>"` is duplicating in 2 messages, create constant